packages/clerk-js/src/utils/memoizeStateListenerCallback.ts

```diff
@@ -106,6 +107,7 @@ function stateWithMemoizedResources(cur: Resources, next: Resources): Resources
       next.lastOrganizationInvitation,
     ),
     lastOrganizationMember: getSameOrUpdatedResource(cur.lastOrganizationMember, next.lastOrganizationMember),
+    authStateKey: getSameOrUpdatedResource(cur.authStateKey, next.authStateKey),
```

Comment:
Can you explain why this is needed?

Reply:
@nikosdouvlis From what I understand this is needed to notify the listener that the resource has been changed so it can get the new value. Without this, the authStateKey just has the initial value(`' '`) and never changes after that.